android/build.gradle

```diff
@@ -0,0 +1,35 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    ext.kotlin_version = "1.3.31"
+    ext.rxAndroidVersion = "2.1.0"
+    ext.rxKotlinVersion = "2.3.0"
+    ext.okhttp3Version = "3.12.0"
+    ext.moshiVersion = "1.8.0"
+    ext.kotlinCoroutineVersion = "1.2.1"
+    ext.ktlintVersion = "0.32.0"
+
+    repositories {
+        google()
+        jcenter()
```

Comment:
If we can remove jcenter, that would be best. We can try tomorrow.

Reply:
`koptional` needs it. I'll delete it once we remove koptional